EditorDelegates/EditorDelegatesUser.cs

```diff
@@ -0,0 +1,52 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+
+namespace Unity.Labs.SuperScience
```

Comment:
Is it possible to include some kind of functionality (maybe through the same delegates system) to automatically open the delegates window when playmode enters/the object activates?  One less step for the user to do to start seeing the results.

Reply:
Great suggestion. I will do that.